admin/sql/updates/20200630-mbs-10921-unlist-unrelated-edits.sql

```diff
@@ -0,0 +1,286 @@
+        SELECT track.recording
+        FROM track
+        JOIN medium ON track.medium = medium.id
+        WHERE medium.release = er.release
+    )
+    LIMIT 10000 -- of 1070983
+)
+RETURNING *;
+
+\echo MBS-10921 (10/12): Unlist “Edit relationship” edits from releases’ history...
+DELETE FROM edit_release
+WHERE (edit, release) IN (
+    SELECT er.edit, er.release
+    FROM edit_release er
+    JOIN edit e ON er.edit = e.id
+    JOIN edit_data ed ON e.id = ed.edit
+    WHERE e.id >= 70658947 AND e.id < 71006471
+    AND e.type = 91
+    AND jsonb_extract_path_text(ed.data, 'type0') = 'recording'
+    AND jsonb_extract_path_text(ed.data, 'type1') = 'work'
+    AND jsonb_extract_path_text(ed.data, 'link', 'entity0', 'id')::int IN (
+        SELECT id FROM recording
+    )
+    AND jsonb_extract_path_text(ed.data, 'link', 'entity0', 'id')::int NOT IN (
+        SELECT track.recording
+        FROM track
+        JOIN medium ON track.medium = medium.id
+        WHERE medium.release = er.release
+    )
+    LIMIT 5000 -- of 512854
+)
+RETURNING *;
+
+\echo MBS-10921 (11/12): Unlist “Remove relationship” edits from releases’ history...
+DELETE FROM edit_release
+WHERE (edit, release) IN (
+    SELECT er.edit, er.release
+    FROM edit_release er
+    JOIN edit e ON er.edit = e.id
+    JOIN edit_data ed ON e.id = ed.edit
+    WHERE e.id >= 70658947 AND e.id < 71006471
+    AND e.type = 92
+    AND jsonb_extract_path_text(ed.data, 'relationship', 'link', 'type', 'entity0_type') = 'recording'
+    AND jsonb_extract_path_text(ed.data, 'relationship', 'link', 'type', 'entity1_type') = 'work'
+    AND jsonb_extract_path_text(ed.data, 'relationship', 'entity0', 'id')::int IN (
+        SELECT id FROM recording
+    )
+    AND jsonb_extract_path_text(ed.data, 'relationship', 'entity0', 'id')::int NOT IN (
+        SELECT track.recording
+        FROM track
+        JOIN medium ON track.medium = medium.id
+        WHERE medium.release = er.release
+    )
+    LIMIT 250 -- of 23395
+)
+RETURNING *;
+
+\echo MBS-10921 (12/12): Unlist “Reorder relationships” edits from releases’ history...
+DELETE FROM edit_recording
+WHERE (edit, recording) IN (
+    SELECT er.edit, er.release
+    FROM edit_release er
+    JOIN edit e ON er.edit = e.id
+    JOIN edit_data ed ON e.id = ed.edit
+    WHERE e.id >= 70658947 AND e.id < 71006471
+    AND e.type = 99
+    AND jsonb_extract_path_text(ed.data, 'link_type', 'entity0_type') = 'recording'
+    AND jsonb_extract_path_text(ed.data, 'link_type', 'entity1_type') = 'work'
+    AND jsonb_extract_path_text(data, 'relationship_order', '0', 'relationship', 'entity1', 'id')::int IN (
+        SELECT id FROM recording
+    )
+    AND jsonb_extract_path_text(data, 'relationship_order', '0', 'relationship', 'entity1', 'id')::int NOT IN (
+        SELECT track.recording
+        FROM track
+        JOIN medium ON track.medium = medium.id
+        WHERE medium.release = er.release
+    )
```

Comment:
Same here re handling every relationship_order item and s/entity1/entity0/.

Reply:
Thanks!

Reply:
> Same here re handling every relationship_order item

Unneeded, same as https://github.com/metabrainz/musicbrainz-server/pull/1581#discussion_r448609981.

> and s/entity1/entity0/.

Fixed in https://github.com/metabrainz/musicbrainz-server/pull/1581/commits/ea6b10dd7562bed837af8b73ae7a49705bc12337.